frontend_server/test/server_test.dart

```diff
@@ -1,19 +0,0 @@
-import 'dart:async';
```

Comment:
Why is this test being removed?

Reply:
It didn't seem like it had been running.

Reply:
Does it pass? Would it be valuable to have running somewhere? Maybe @aam knows.

Reply:
It's not high-value test, other build-time things will break if running with `--train` fails somehow. Also there used to be more tests as part of this, but they were migrated over to dart sdk. I would keep it, but no strong feelings if it is deleted.